### PR TITLE
Upgrade version of Microsoft.Metadata.Visualizer

### PR DESCRIPTION
**Pull request by 333fred**
Fixes https://github.com/dotnet/metadata-tools/issues/30
Adds support for calli token printing

### CODE REVIEW

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs [hunk 1]
```diff
@@ -5924,7 +5924,7 @@ .maxstack  2
   IL_003a:  brfalse.s  IL_0042
   IL_003c:  ldloc.0
   IL_003d:  call       ""void System.Console.Write(bool)""
-  IL_0042:  ldc.r8     Infinity
+  IL_0042:  ldc.r8     ∞
```
Comment (tmat) [inline src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs]: Nice :)

File: src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenConstLocal.vb [hunk 2]
```diff
@@ -181,7 +181,7 @@ End Module
 
         End Sub
 
-        <Fact()>
```
Comment (tmat) [inline src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenConstLocal.vb]: Rather then skipping this test, shouldn't we use platform agnostic raw display?

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs (hunk 1 above)
Comment (tannergooding) [reply src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs]: Is this going to cause problems when run on non "en" cultures (seems the invariant culture isn't being used)?

File: src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenConstLocal.vb (hunk 2 above)
Comment (333fred) [reply src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenConstLocal.vb]: @tmat this ended up being an issue where the test output itself was not using G15. Using G15 ensures that everything works as expected, and I've re-enabled the tests.

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs (hunk 1 above)
Comment (333fred) [reply src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs]: I've filled https://github.com/dotnet/metadata-tools/issues/44 to track. I'm not going to hold up this PR anymore, I need to get it in to address some testing issues in my function-pointers branch.

**Comment by AlekseyTs** [comment]
@333fred It looks like CI jobs are failing

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs (hunk 1 above)
Comment (AlekseyTs) [reply src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs]: > I've filled [https://github.com/dotnet/metadata-tools/issues/44](https://github.com/dotnet/metadata-tools/issues/44) to track. I'm not going to hold up this PR anymore, I need to get it in to address some testing issues in my function-pointers branch.

Could we run into issues with Spanish CI job?

---
In reply to: [396761882](https://github.com/dotnet/roslyn/pull/42641#discussion_r396761882) [](ancestors = 396761882)

**Comment by AlekseyTs** [comment]
Done with review pass (iteration 3)

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs (hunk 1 above)
Comment (333fred) [reply src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs]: Turns out, yes we can. PR'd https://github.com/dotnet/metadata-tools/issues/44 to fix

**Comment by AlekseyTs** [comment]
@333fred It looks like there are legitimate test failures

**Comment by AlekseyTs** [comment]
@333fred It looks like CI is still broken

**Pull request merged**